portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/moresites.vm

```diff
@@ -144,20 +147,19 @@
             <div class="moresites-right-col">
               #foreach( $termKey in $tabsSites.tabsMoreSortedTermList )
                 #if ($tabsSites.tabsMoreTermsRightPane.get($termKey).size() > 0)
-                  <div class="fav-sites-term">
-                    #if ( $termKey && $termKey != "" )
-                      <h3 class="favorites-term-header">
-                        <span class="favorites-select-container">
-                          <button class="favorites-select-all-none icon-button"
-                              title="$rloader.getFormattedMessage("moresite_toggle_all_as_favorites", $termKey)"
-                              aria-label="$rloader.getFormattedMessage("moresite_toggle_all_as_favorites", $termKey)">
-                            <i class="bi-pin-fill" style="display: none"></i>
-                            <i class="bi-pin" style="display: none"></i>
-                          </button>
-                        </span>
+                  <div class="fav-sites-card card mb-2">
+                    <div class="card-header">
+                      <h3 class="card-title">
                         <span>$termKey</span>
+                        <button class="favorites-select-all-none icon-button ms-1"
+                            title="$rloader.getFormattedMessage("moresite_toggle_all_as_favorites", $termKey)"
+                            aria-label="$rloader.getFormattedMessage("moresite_toggle_all_as_favorites", $termKey)">
+                          <i class="si si-pin-fill" style="display: none"></i>
+                          <i class="si si-pin" style="display: none"></i>
```

Comment:
```suggestion
                          <i class="si si-pin-fill d-none"></i>
                          <i class="si si-pin d-none"></i>
```

Reply:
You need to update the js too. These are toggled in js.